kettle/Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime \
 RUN pip3 install requests google-cloud-pubsub==0.25.0 google-cloud-bigquery==0.27.0 influxdb ruamel.yaml==0.16
 
 RUN curl -fsSL https://downloads.python.org/pypy/pypy3.6-v7.3.1-linux64.tar.bz2 | tar xj -C opt  && \
-    ln -s /opt/pypy*/bin/pypy /usr/bin
+    ln -s /opt/pypy*/bin/pypy3 /usr/bin
```

Comment:
I couldn't find anything in release notes for 7.1 - 7.3 that would explain this, super weird

Reply:
neither could I, and I have tried it in a few env and still only get `pypy3`